schemas/com.snowplowanalytics.snowplow/web_vitals/jsonschema/1-0-0

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a web vitals tracking event. For more information on web vitals you can visit https://web.dev/vitals/.",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "web_vitals",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "cls": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": -2147483647,
+            "description": "A unitless metric for measuring visual stability because it helps quantify how often users experience unexpected layout shifts. For more information https://web.dev/cls/."
+        },
+        "fid": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": -2147483647,
+            "description": "A metric for measuring load responsiveness because it quantifies the experience users feel when trying to interact with unresponsive pages. Measured in milliseconds. For more information https://web.dev/fid/."
+        },
+        "lcp": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": -2147483647,
+            "description": "A metric for measuring perceived load speed because it marks the point in the page load timeline when the page's main content has likely loaded. Measured in milliseconds. For more information https://web.dev/lcp/."
+        },
+        "fcp": {
+            "type": ["number", "null"],
+            "maximum": 2147483647,
+            "minimum": -2147483647,
+            "description": "A metric for measuring perceived load speed because it marks the first point in the page load timeline where the user can see anything on the screen. Measured in milliseconds. For more information https://web.dev/fcp/."
+        },
+        "inp": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": -2147483647,
+            "description": "A metric that assesses responsiveness. INP observes the latency of all interactions a user has made with the page, and reports a single value which all (or nearly all) interactions were below that value. For more information https://web.dev/inp/."
+        },
+        "ttfb": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": -2147483647,
+            "description": "A DOMHighResTimeStamp referring to the time in milliseconds between the browser requesting a page and when it receives the first byte of information from the server. For more information https://web.dev/ttfb/."
+        },
+        "navigationType": {
```

Comment:
Nit: Is it intentional that there is camel-case here? I think we informally agreed on the underscore case for new schemas if there is no previous context where other case is used. (btw, this is a tiny nitpick)